### PR TITLE
Add osu! Philippines: CONQuest PH 2022 Newspost

### PR DESCRIPTION
**Pull request by topecnz**
Written by me, @sr229, and @LeNitrous 
Reviewed by: Osu Tatakae Ouendan and ToGlette

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

Co-Authored-By: Ayane <ayane@vignetteapp.org>
Co-Authored-By: Nathan Alo <nitrous@vignetteapp.org>

### CODE REVIEW

**Review by sr229 — COMMENTED** [review]
Just a nit

File: news/2022-07-21-osu-philippines-conquest-ph-2022.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "osu! Philippines: CONQuest PH 2022"
+date: 2022-07-21 00:00:00 +0000
+---
+
+A monumental first for osu! in the Philippines, an osu!booth will be open at [CONQuest PH 2022](https://conquestph.com), held at the SMX Convention Centre, Pasay City, Philippines.
+
+The first ever LAN event, the **Quickfire Showdown**, will be hosted at the booth!
```
Comment (TPGPL) [inline news/2022-07-21-osu-philippines-conquest-ph-2022.md:9]:
```suggestion
The first ever Filipino LAN event, the **Quickfire Showdown**, will be hosted at the booth! 
```

more correct this way i guess?

File: news/2022-07-21-osu-philippines-conquest-ph-2022.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+Additionally, as a CONQuest PH 2022 exclusive, there will be a track from one of osu!'s featured artists, [LEAF XCEED Music Division](https://osu.ppy.sh/beatmaps/artists/88)! 
+
+## Prizes
+
+### July 23, 2022 (Day 1)
+
+| Placing | Prizes |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | Profile badge + osu!merch + 3 months of osu!supporter |
+| Runner-up | osu!merch |
```
Comment (TPGPL) [inline news/2022-07-21-osu-philippines-conquest-ph-2022.md:28]: might wanna use the silver crown for runner-up?

File: news/2022-07-21-osu-philippines-conquest-ph-2022.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+Additionally, as a CONQuest PH 2022 exclusive, there will be a track from one of osu!'s featured artists, [LEAF XCEED Music Division](https://osu.ppy.sh/beatmaps/artists/88)! 
+
+## Prizes
+
+### July 23, 2022 (Day 1)
+
+| Placing | Prizes |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | Profile badge + osu!merch + 3 months of osu!supporter |
```
Comment (TPGPL) [inline news/2022-07-21-osu-philippines-conquest-ph-2022.md:27]:
```suggestion
| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, osu!merch, 3 months of osu!supporter |
```

File: news/2022-07-21-osu-philippines-conquest-ph-2022.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+| Runner-up | osu!merch |
+
+### July 24, 2022 (Day 2)
+
+| Placing | Prizes |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | Profile badge + osu!merch + 3 months of osu!supporter |
```
Comment (TPGPL) [inline news/2022-07-21-osu-philippines-conquest-ph-2022.md:34]:
```suggestion
| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, osu!merch, 3 months of osu!supporter |
```

File: news/2022-07-21-osu-philippines-conquest-ph-2022.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+| :-: | :-- |
+| Registration phase | 10:00 - 13:00 UTC+8 |
+| Round of 16 | 13:00 - 15:00 UTC+8 |
+| Quarterfinals | 15:00 - 16:00 UTC+8 |
+| Semifinals | 16:00 - 17:00 UTC+8 |
+| Finals | 17:00 UTC+8 |
+
+## Booth Activties
+
+Aside from the tournament, other activities will also be held at the booth, such as a player hub for fellow community members to meet up. A free play area with devices running osu!lazer will also be open for those both new and familiar with osu! to try out osu!lazer. osu!lazer will be preloaded with tracks from the [featured artists](https://osu.ppy.sh/beatmaps/artists).
```
Comment (TPGPL) [inline news/2022-07-21-osu-philippines-conquest-ph-2022.md:61]:
```suggestion
Aside from the tournament, other activities will also be held at the booth, such as a player hub for fellow community members to meet up. A free play area with devices running osu!lazer will also be open for those both new and familiar with osu! to try out. osu!lazer will be preloaded with tracks from the [featured artists](https://osu.ppy.sh/beatmaps/artists). 
```

File: news/2022-07-21-osu-philippines-conquest-ph-2022.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+| Semifinals | 16:00 - 17:00 UTC+8 |
+| Finals | 17:00 UTC+8 |
+
+## Booth Activties
+
+Aside from the tournament, other activities will also be held at the booth, such as a player hub for fellow community members to meet up. A free play area with devices running osu!lazer will also be open for those both new and familiar with osu! to try out osu!lazer. osu!lazer will be preloaded with tracks from the [featured artists](https://osu.ppy.sh/beatmaps/artists). 
+
+For our international audience or those who couldn't attend the convention, you can stay in the loop by watching the stream at [osu! Philippines Twitch Channel](https://twitch.tv/osuliveph).
+
+See you in-stream and in-person!
```
Comment (TPGPL) [inline news/2022-07-21-osu-philippines-conquest-ph-2022.md:65]:
```suggestion
See you on-stream and in-person!
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: news/2022-07-21-osu-philippines-conquest-ph-2022.md (hunk 2 above)
Comment (topecnz) [reply news/2022-07-21-osu-philippines-conquest-ph-2022.md:28]: looks good, sure!

**Review by topecnz — COMMENTED** [review]
All good I guess.

File: news/2022-07-21-osu-philippines-conquest-ph-2022.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "osu! Philippines: CONQuest PH 2022"
+date: 2022-07-21 00:00:00 +0000
+---
+
+A monumental first for osu! in the Philippines, an osu!booth will be open at [CONQuest PH 2022](https://conquestph.com), held at the SMX Convention Centre, Pasay City, Philippines.
```
Comment (peppy) [inline news/2022-07-21-osu-philippines-conquest-ph-2022.md:7]:
```suggestion
A monumental first for osu! community in the Philippines, an (unofficial) osu! community run space will be open at [CONQuest PH 2022](https://conquestph.com), held at the SMX Convention Centre, Pasay City, Philippines.
```

File: news/2022-07-21-osu-philippines-conquest-ph-2022.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+| :-: | :-- |
+| Registration phase | 10:00 - 13:00 UTC+8 |
+| Round of 16 | 13:00 - 15:00 UTC+8 |
+| Quarterfinals | 15:00 - 16:00 UTC+8 |
+| Semifinals | 16:00 - 17:00 UTC+8 |
+| Finals | 17:00 UTC+8 |
+
+## Booth Activties
+
+Aside from the tournament, other activities will also be held at the booth, such as a player hub for fellow community members to meet up. A free play area with devices running osu!lazer will also be open for those both new and familiar with osu! to try out. osu!lazer will be preloaded with tracks from the [featured artists](https://osu.ppy.sh/beatmaps/artists).
```
Comment (peppy) [inline news/2022-07-21-osu-philippines-conquest-ph-2022.md:61]:
```suggestion
Aside from the tournament, other activities will also be held at the booth, such as a player hub for fellow community members to meet up. A free play area with devices running osu!(lazer) will also be open for those both new and familiar with osu! to try out. 
```

Not sure why mentioning featured artists is required.

File: news/2022-07-21-osu-philippines-conquest-ph-2022.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+| Round of 16 | 13:00 - 15:00 UTC+8 |
+| Quarterfinals | 15:00 - 16:00 UTC+8 |
+| Semifinals | 16:00 - 17:00 UTC+8 |
+| Finals | 17:00 UTC+8 |
+
+## Booth Activties
+
+Aside from the tournament, other activities will also be held at the booth, such as a player hub for fellow community members to meet up. A free play area with devices running osu!lazer will also be open for those both new and familiar with osu! to try out. osu!lazer will be preloaded with tracks from the [featured artists](https://osu.ppy.sh/beatmaps/artists). 
+
+For our international audience or those who couldn't attend the convention, you can stay in the loop by watching the stream at the [osu! Philippines Twitch channel](https://twitch.tv/osuliveph).
```
Comment (peppy) [inline news/2022-07-21-osu-philippines-conquest-ph-2022.md:63]:
```suggestion
For those who can't attend the convention but are interested in following along, you can stay in the loop by watching live-streamed content at the [osu! Philippines Twitch channel](https://twitch.tv/osuliveph).
```

**Review by peppy — CHANGES_REQUESTED** [review]

**Review by peppy — DISMISSED** [review]

**Comment by peppy** [comment]
Looks good from my end. Deferring to @Ephemeralis for scheduling.

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**